schemas/com.snowplowanalytics.snowplow/parent_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Context containing parent event id",
```

Comment:
Change to "Context containing ID of parent event (useful for annotating events derived from other events)"